CPDShell/Core/scrubber/abstract_scrubber.py

```diff
@@ -0,0 +1,63 @@
+from abc import ABC, abstractmethod
+from collections.abc import Iterable, Sequence
+
+import numpy
+
+from CPDShell.Core.scenario import Scenario
+
+
+class AbstractScrubber(ABC):
+    """A scrubber for dividing data into windows
+    and subsequent processing of data windows
+    by change point detection algorithms
+    """
+
+    def __init__(self) -> None:
+        """A scrubber for dividing data into windows
+        and subsequent processing of data windows
+        by change point detection algorithms
+
+        """
+        self._scenario: Scenario | None = None
```

Comment:
Should we include `Scenario` type in alpha version? We don't have any other scrubbers for now. It overcomplicates algorithm construction, making one more layer.